executor/examples/exec_block.rs

```diff
@@ -80,6 +81,9 @@ async fn main() -> anyhow::Result<()> {
     //let mut cache_db = CacheDB::new(ethersdb);
     let mut cache_db = CacheDB::new(EmptyDB::default());
     // get pre
+
+    let mut db = StateDB::new(None);
```

Comment:
We should not make the DB directly here, we have to support different DBs, like a local db for unit test.